### PR TITLE
Update build & test tooling

### PR DESCRIPTION
**Pull request by jnyrup**
All changes in the first commit are from running `nuke :update` with Nuke.GlobalTool.

### CODE REVIEW

File: Build/Build.cs [hunk 1]
```diff
@@ -27,13 +28,11 @@ class Build : NukeBuild
         x => x.UnitTests,
         x => x.Pack);
 
-    [Solution] readonly Solution Solution;
+    [Solution(GenerateProjects = true)] readonly Solution Solution;
```
Comment (jnyrup) [inline Build/Build.cs:31]: This property enables the generation of `Solution.` properties.

File: Src/FluentAssertions/FluentAssertions.csproj [hunk 2]
```diff
@@ -5,6 +5,7 @@
     <AssemblyOriginatorKeyFile>FluentAssertions.snk</AssemblyOriginatorKeyFile>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>1591;1573</NoWarn>
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
```
Comment (jnyrup) [inline Src/FluentAssertions/FluentAssertions.csproj:8]: Using a .NET 5 compiler introduces a warning about EOL Frameworks such as netcoreapp2.0 and netcoreapp3.0

File: Tests/Approval.Tests/ApiApproval.cs [hunk 3]
```diff
@@ -25,7 +25,8 @@ public class ApiApproval
         [InlineData("netcoreapp3.0")]
         public Task ApproveApi(string frameworkVersion)
         {
-            string codeBase = Assembly.GetExecutingAssembly().CodeBase;
+            VerifierSettings.DisableClipboard();
+            string codeBase = Assembly.GetExecutingAssembly().Location;
```
Comment (jnyrup) [inline Tests/Approval.Tests/ApiApproval.cs:29]: To resolve deprecation/obsoletion warnings

**Review by dennisdoomen — APPROVED** [review]
❤

**Pull request merged**